pages/index.tsx

```diff
@@ -52,8 +52,6 @@ export default function Donate() {
     return () => window.removeEventListener('wheel', handleScroll);
   });
   return (
-    <Layout>
-      {projects ? <ProjectsList {...DonateProps} /> : <h2>Loading...</h2>}
-    </Layout>
+    <Layout>{projects ? <ProjectsList {...DonateProps} /> : <h2></h2>}</Layout>
```

Comment:
can we make the screen color same as map blue when this loads?
or instead load the project loader here?

Reply:
I am sure this is done in fix scroll PR. the color is not accurate though I have messaged Killian to get the actual color.

Reply:
also, scroll PR should have all the changes in this PR so instead merge that and close this PR.